Improve metadata output (#112)
echemdb/svgdigitizer: doc/news/metadataoutput.rst

**Added:**

> * Added the property `simultaneous_measurement` and `comment` to `svgdigitizer.electrochemistry.cv`, which read these properties from a text field in the SVG file.

Reviewer: ```suggestion
* Added the property `svgdigitizer.electrochemistry.cv.CV.simultaneous_measurement` which is read from a text field in the SVG file.
```

I don't think you added comment.

**Changed:**

> * Changed the output of the property `metadata` in `svgdigitizer.electrochemistry.cv`. It includes 'data_description', which contains the units to the values in the dataframe (df).

Reviewer: ```suggestion
* Changed the property `svgdigitizer.electrochemistry.cv.CV.metadata`. It now includes 'data description' which contains the units to the values in the dataframe.
```

> * Changed the dimension on the x-axis from `U` to `E` (potential).

Reviewer: ```suggestion
* Changed the dimension on the x-axis of CVs from `U` to `E` (potential).
```
